### ISSUE TITLE
Update FF Meta information to show Sign-up

### ISSUE THREAD

**Issue by iskerrett** [FlowFuse/website#491]
### Description

When someone searches on FlowForge, Google will show some of the key links we want to highlight. I don't exactly know where this is done but I believe it is in the website meta information.  It would be nice to update this to include a Sign-up to FlowForge. 

We should also update the About and Docs description to be something like:
About: FlowForge is the company behind Node-RED. 
Docs: Technical documentation on install and using the FlowForge platform.

![image](https://user-images.githubusercontent.com/36051834/222550550-fd289ac5-fc42-47d2-9bcd-8b6b94a77db3.png)

**Comment by joepavitt** [comment]
[According to Google](https://developers.google.com/search/docs/appearance/sitelinks), these are automated, and we can't explicitly set them.

However, some takeaway points to consider from your request:

- There is a ["Sitemap Priority"](https://firespring.my.salesforce-sites.com/nonprofit/articles/FAQ/The-XML-Sitemap-and-Sitemap-Priority#:~:text=Sitemap%20Priority%20refers%20to%20the,is%20Normal%2C%20for%20all%20pages.) available for each of our pages, which is related to the ranking of importance. I would suspect that Google rely on this heavily, even though they don't explicitly state that. I made the priority a configurable value [here](https://github.com/flowforge/website/pull/297) so we can re-visit those (looking in our auto-generated sitemap.xml) to understand if we want to change any of those.
- We don't currently have a "Sign Up' page indexed on our Website, that's hosted on our Application, which wouldn't show here as a site link. We could/should have a "Sign Up" page that is formatted like the sign up page on app, but just has a single button called "Sign Up' - which then directs users to the App itself?
- `FlowForge is the company behind Node-RED.` - this is not strictly, true, and a confusion that I know @knolleary was keen avoid in the past. Whilst we are building around Node-RED, and want to be known as a key player with respect to Node-RED, we are not responsible for it, nor do we own it, and I feel that saying we are "behind node-RED" may be a little mis-leading as it's an Open Source project, under theO OpenJS Foundation.
- Interestingly, it's taking the "FlowForge adds teams..." from the first paragraph it's finding on our homepage, which our tagline, I've currently rendered as a <h3> - this would be updated, so that Google grabs the tagline ("FlowForge allows organisations to reliably deliver Node-RED applications in a continuous, collaborative, and secure manner."). Similarly, it's doing this for the Docs and "Installing" pages too I believe.

**Comment by joepavitt** [comment]
Speaking with Ian, still want to have this actioned, I see one of two options:

1. We move the "sign up" form completely out of the app, and host it on the website - I suspect this would not be a trivial piece of development work?
2. We do the second bullet point above, in that we have a pseudo-sign up screen, which has a button to redirect to the actual sign up page.

**Comment by ZJvandeWeg** [comment]
I think the following works as a first iteration:

```md
---
# Inspired by https://github.com/11ty/eleventy/issues/510#issuecomment-824104799
#
# This file generates redirects for pages that have moved.
#
pagination:
  data: redirects
  size: 1
  alias: redirect
redirects:
  - { "from": "/sign-up", "to": "https://app.flowforge.com/account/create"}
permalink: "{{ redirect.from }}"
# The "redirect" layout just has a small html header with the meta tags that do redirection.
layout: redirect
title: "Create an account for FlowForge"
---
(the content can be left blank; it's entirely the frontmatter doing the work here.)

```

**Comment by joepavitt** [comment]
> I think the following works as a first iteration:

Isn't Google anti-redirects though?

**Comment by Yndira-FlowForge** [comment]
If I understand correctly what they say [here](https://developers.google.com/search/docs/crawling-indexing/301-redirects#metarefresh) it will index `https://app.flowforge.com/account/create`, although I'm still not sure if it'll show it as a sitelink...

And if we wanted `/sign-up` indexed, which I think would increase the chance of it showing as a site link, we should delay the redirect, but I couldn't find the minimum time required to be considered a temporary redirect by Google. And seems like eventually, even with the delay, Google will stop considering it temporary and will index `https://app.flowforge.com/account/create` instead.

Will dig deeper and keep you posted.

**Comment by gstout52** [comment]
> * We don't currently have a "Sign Up' page indexed on our Website, that's hosted on our Application, which wouldn't show here as a site link. We could/should have a "Sign Up" page that is formatted like the sign up page on app, but just has a single button called "Sign Up' - which then directs users to the App itself?

I'd like to raise this as possibly the easiest-to-implement solution. @joepavitt @Yndira-E Can we put up a basic sign-up page on the website that has no hyperlinks to it from the rest of the website, and that links to the account creation page in app.flowfuse.com?

**Comment by joepavitt** [comment]
Just to clarify, we'd still have the signup form in `app.flowfuse.com` - but you just want a placeholder page `/signup` on the website that links to this? 

Where would our CTAs link to? Ideally, we'd host the sign up page in the Website, but it's not "easy"

**Comment by Yndira-E** [comment]
I think Greg means leaving our CTAs as they are currently, and simply adding a `/signup` page on the website with a single CTA linking to `app.flowfuse.com/account/create/`. This page would serve as a landing page to be listed as a sitelink in search results, and the only ways to reach it would be either through search results or by typing the URL directly. It wouldn't be a functional sign-up form itself, just a bridge page directing users to the actual sign-up flow in the app. Although I'm not sure what the content would be for this page.

**Comment by joepavitt** [comment]
Yeah, and also, it wouldn't necessarily get indexed as a high priority page due to lack of interaction/visitors and links to the page

**Comment by gstout52** [comment]
> Just to clarify, we'd still have the signup form in `app.flowfuse.com` - but you just want a placeholder page `/signup` on the website that links to this?
> 
> Where would our CTAs link to? Ideally, we'd host the sign up page in the Website, but it's not "easy"

Well, if the page won't be indexed as high priority because it's just a placeholder, then it sounds like that would not be a valuable first iteration.
In that case, let's evaluate the work needed to move the signup page to the website. If it's a very big lift, it doesn't need to be top priority.

**Comment by ZJvandeWeg** [comment]
> Yeah, and also, it wouldn't necessarily get indexed as a high priority page due to lack of interaction/visitors and links to the page

Either way, why can't we just fix the canonical and get it indexed?

**Comment by Yndira-E** [comment]
I've been looking for ways to achieve this, and it seems like adding it to our sitemap should increase our chances of having it listed as a sitelink. [Here's a PR](https://github.com/FlowFuse/website/pull/2902) that does that.

Additionally, adding a meta title and meta description to https://app.flowfuse.com/account/create would help.

**Comment by Yndira-E** [comment]
Meta title could be: Create Your FlowFuse Account
And meta description: Sign up for FlowFuse and start building applications, integrating data, and managing your Node-RED instances with enterprise-grade security.

**Comment by Yndira-E** [comment]
I noticed the sign up page has `<link rel="canonical" href="https://app.flowfuse.com/" />`  Should it be changed to `<link rel="canonical" href="https://app.flowfuse.com/account/create" />`?

**Comment by ZJvandeWeg** [comment]
Yes, but the engineers didn't properly set it

**Comment by cstns** [comment]
The canonical link is injected application wide in `forge/routes/ui/index.js:81` as the `base_url` which makes sense as it's an SPA and there are no pages to navigate to.

In order for any crawler to index the auth pages (login, create/subscribe, reset pw, etc) correctly, they'd need to be served statically not dynamically as they are now, usually through server side rendering.

I'm looking into [SSR](https://vuejs.org/guide/scaling-up/ssr.html#why-ssr) & [SSG](https://vuejs.org/guide/scaling-up/ssr.html#ssr-vs-ssg) and see how we could deploy the auth pages within our current setup

**Comment by ZJvandeWeg** [comment]
As a first iteration we've removed the wrong canonical to hopefully get indexed again.

**Comment by cstns** [comment]
First train of thought on how to get ssr on auth pages was to break up a chunk of the app similarly to the setup stage and have it run ssr only. Having tinkered a bit with the idea this is what I got out of it:
- seo wise, only the initial page load matters, what happens afterwards does not matter
- kind of messy, would imply a lot of changes
- would need to create a distinct app from a chunk of the main app while also leaving it in the main part (the auth pages)
  - use the auth pages into separate app that are server side rendered when accessing link/route directly
  - leave same part of the app as an integral component of the main spa so regular flow is maintained when users logout/log back in (what you do after logging out shouldn't really matter if you're redirected by the vue router on a dynamic route or to a ssr page)
- would need to create endpoints for each auth endpoint (pw recovery, login, signup etc)

Looking into cleaner solutions with 3rd party frameworks
